src/verilog.y

```diff
@@ -6930,6 +6930,11 @@ covergroup_declarationFront<classp>:  // IEEE: part of covergroup_declaration
                           BBCOVERIGN($<fl>1, "Ignoring unsupported: covergroup"); }
                 ;
 
+cgPortList<nodep>:
+        /*empty*/                       { $$ = nullptr; }
+        |       '(' tf_port_listE ')'   { $$ = $2;      }
```

Comment:
Add "E" meaning it can be empty.  Check `{` line up with column 56.

```suggestion
cgPortListE<nodep>:
                /*empty*/             { $$ = nullptr; }
        |       '(' tf_port_listE ')'   { $$ = $2;      }
```